src/navbar/types.ts

```diff
@@ -60,10 +68,25 @@ export type DrawerDismissalMethod =
 export interface NavbarDrawerProps extends NavbarSharedProps {
     show: boolean;
     children: JSX.Element | JSX.Element[];
-    onClose?: (() => void) | undefined;
+    onClose?: () => void | undefined;
+    resourcesSecondaryBrand?: NavbarResourcesProps | undefined;
+    handleSecondaryBrandClick?: (
```

Comment:
onSecondaryBrandClick

Reply:
on side note, should merge into `onBrandClick` but that returns a param to tell whether is it the pri or secondary brand that was clicked